ecosystem/scenarios/utils.py

```
@@ -0,0 +1,254 @@
+
+    result = subprocess.run(cmd, capture_output=True, text=True)
+
+    if result.returncode != 0:
+        print(f"Cribo failed with exit code {result.returncode}")
+        print(f"STDOUT:\n{result.stdout}")
+        print(f"STDERR:\n{result.stderr}")
+
+    return result
+
+
+def run_bundled_test(bundled_path: str, test_script: str) -> subprocess.CompletedProcess:
```

Pylint (3.3.7)

[warning] Unused argument 'bundled_path'

(W0613)